java/ingest/ingest-core/src/main/java/sleeper/ingest/job/IngestJobValidationUtils.java

```diff
@@ -37,16 +42,20 @@ public static <T> Optional<T> deserialiseAndValidate(
         T job;
         try {
             job = deserialiser.apply(message);
+            LOGGER.info("Job deserialised successfully");
         } catch (RuntimeException e) {
+            LOGGER.info("Deserialisation failed: {}", e.getMessage());
```

Comment:
Maybe this should be at warn or error level? Maybe we should log the full stack trace if the exception is worth logging?